### DISCUSSION TITLE
Truncate or locally suppress inlay hints

### DISCUSSION THREAD

**Discussion by lukasjuhrich** [zigtools/zls#1924] [Q&A] [answered]
Hi y'all,

This is my first time interacting with zig and the `zls`. I've noticed that in some scenarios, like many error variants, the inlay hints become absolutely huge, e.g. when using `sty.fs.realpath`:
![image](https://github.com/zigtools/zls/assets/10060406/5ea1fd08-98d6-4277-9c89-1efc9006334e)

This is quite disruptive, however I really don't want to disable inlay hints. I know ZLS offers some options, but none of them seem to help me here.

Is there anything I can do to ameliorate this?

**Comment by techatrix** [comment, marked as answer]
This is a known issue which has been reported in https://github.com/zigtools/zls/issues/1828

**Reply by lukasjuhrich** [reply]
Thanks, not sure why I didn't find that issue.
